### ISSUE TITLE
add file as part of Lab IT SSL cert generation for Cloudflare configuration

### ISSUE THREAD

**Issue by dehays** [microbiomedata/nmdc-server#500]
@subdavis  Can you add this file with the specified content:

> Sometimes it takes a few minutes to auto-provision a Cloudflare edge SSL certificate that would be used to communicate with clients.
> We can validate and pre-provision by creating:
> http://data.microbiomedata.org/.well-known/pki-validation/ca3-d4874059fae949469e9264af71f4874e.txt
> 
> with content 
> 
> ca3-f4ec159bc50e4ad495d6065e4b647158

**Comment by subdavis** [comment]
I believe this instruction is coming from https://developers.cloudflare.com/ssl/ssl-for-saas/certificate-validation-methods, is that right?

I'm not sure this is the right way for us to prove ownership.

1. any update to production will break production because the database schema has changed.  I'd have to branch off a commit from several months ago and bypass our existing CI to deploy this as a patch of an old revision.
1. We've already provisioned SSL, so I'd have to reconfigure the load balancer to disable https.
1. I don't think embedding proof-of-ownership documents in our container images is a best practice

Under the "Other" section, they describe our use case (minus the downtime toleration)

> For your customers that already have HTTPS on their custom domain, e.g., they’re self-hosted or you’ve manually provisioned, and cannot tolerate a couple minutes of downtime during cutover, additional “pre-validation” methods are available.

Instead of setting up a static file in the container, could you follow one of the alternative methods, such as `TXT` validation https://developers.cloudflare.com/ssl/ssl-for-saas/certificate-validation-methods#1-txt-record or `CNAME` validation?

**Comment by dehays** [comment]
@subdavis I would agree and I have conveyed this to Lab IT.  I find it amusing that I will be asking the same people (Lab IT) to modify the DNS record to provide them adequate proof that we control the domain in order to generate another SSL certificate.

**Comment by aclum** [comment]
@shreddd  is this still needed?
